src/Symfony/Component/Serializer/Tests/Mapping/Loader/XmlFileLoaderTest.php

```diff
@@ -80,6 +80,8 @@ public function testSerializedName()
         $attributesMetadata = $classMetadata->getAttributesMetadata();
         $this->assertEquals('baz', $attributesMetadata['foo']->getSerializedName());
         $this->assertEquals('qux', $attributesMetadata['bar']->getSerializedName());
+        $this->assertEquals('duxi', $attributesMetadata['duux']->getSerializedName());
+        $this->assertEquals('duxa', $attributesMetadata['duux']->getSerializedName(['a']));
```

Comment:
```suggestion
        $this->assertSame('duxi', $attributesMetadata['duux']->getSerializedName());
        $this->assertSame('duxa', $attributesMetadata['duux']->getSerializedName(['a']));
```
Also for the other possible ones in this test class